actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1671,6 +1671,47 @@ def to_model
         self
       end
 
+      # Generate an HTML <tt>id</tt> attribute value.
```

Comment:
Just a small thing. Let's move those two methods to after the `initialize` definition.